docs/applications/add-ssl-certificate.md

```diff
@@ -6,6 +6,7 @@ links:
     quickstart:
     previous:
     next:
+      - /docs/applications/configure-dns-service
```

Comment:
In this topic a note could be added mentioning that before issuing an SSL certificate a DNS record must be configured pointing to the server on which the application is hosted.

We're not documenting how to manage DNS records. We are documenting how to issue SSL certificates and having a valid DNS record is a requirement - for now.

Reply:
Noted, will make changes to the PR

Reply:
Note added, could you review @jfoliveira ?